nextjs-wordles/public/słownik.js

```diff
@@ -1,18 +1,19 @@
-const Dicionary = fetch('/słownik.txt')
-  .then((response) => response.text())
-  .then((text) => text);
-
+const Dictionary = async () => {
```

Comment:
@AndrzejGrabarz Andrzeju, nie trzeba tutaj żadnych async przy importowaniu zwykłego jsona (stąd moja poprzednia sugestia, żeby nie używać txt). Wystarczy prosty import:
https://www.learnbestcoding.com/post/81/3-ways-to-import-a-json-file-in-react

Reply:
Zamienione na import